source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -713,3 +640,60 @@ void TextRunToRichTextElementProperties(const std::shared_ptr<TextRun> &textRun,
     const std::string str = config->GetForegroundColor([ACOHostConfig getSharedContainerStyle:style], textColor, isSubtle);
     return [ACOHostConfig convertHexColorCodeToUIColor:str];
 }
+
+void configVerticalAlignmentConstraintsForBackgroundImageView(const BackgroundImage *backgroundImageProperties, UIView *superView, UIImageView *imageView)
+{
+    if (!backgroundImageProperties || !superView || !imageView) {
+        return;
+    }
+
+    switch (backgroundImageProperties->GetVerticalAlignment()) {
+        case VerticalAlignment::Bottom:
+            [imageView.bottomAnchor constraintEqualToAnchor:superView.bottomAnchor].active = YES;
+            break;
+        case VerticalAlignment::Center:
+            [imageView.centerYAnchor constraintEqualToAnchor:superView.centerYAnchor].active = YES;
+            break;
+        case VerticalAlignment::Top:
+        default:
+            [imageView.topAnchor constraintEqualToAnchor:superView.topAnchor].active = YES;
+            break;
+    }
+}
+
+void configHorizontalAlignmentConstraintsForBackgroundImageView(const BackgroundImage *backgroundImageProperties, UIView *superView, UIImageView *imageView)
+{
+    if (!backgroundImageProperties || !superView || !imageView) {
+        return;
+    }
+
+    switch (backgroundImageProperties->GetHorizontalAlignment()) {
+        case HorizontalAlignment::Right:
+            [imageView.trailingAnchor constraintEqualToAnchor:superView.trailingAnchor].active = YES;
+            break;
+        case HorizontalAlignment::Center:
+            [imageView.centerXAnchor constraintEqualToAnchor:superView.centerXAnchor].active = YES;
+            break;
+        case HorizontalAlignment::Left:
+        default:
+            [imageView.leadingAnchor constraintEqualToAnchor:superView.leadingAnchor].active = YES;
+            break;
+    }
+}
+
+void configWidthAndHeightAnchors(UIImageView *imageView, UIView *superView, bool isComplimentaryAxisHorizontal)
+{
+    if (imageView && imageView.image && superView) {
```

Comment:
>mageView && imageView.image && superView [](start = 9, length = 40)

What's the behavior for !imageView || !imageView.image || !superView? Should you explictly call that case out much like the pattern for the configHorizontal/Vertical routines above for consistent readability?